src/main/target/NAZE/target.h

```diff
@@ -101,15 +101,17 @@
 //#define ACC_MMA8452_ALIGN       CW90_DEG
 //#define ACC_BMA280_ALIGN        CW0_DEG
 
+#if 0
```

Comment:
Can we have this properly commented out, and can we have a comment here please stating why this was commented out?